src/detect-engine-analyzer.c

```diff
@@ -859,6 +859,68 @@ void EngineAnalysisRules2(const DetectEngineCtx *de_ctx, const Signature *s)
     SCReturn;
 }
 
+typedef struct analyzer_item {
+    uint32_t    have_bufcount;
+    int32_t   item_id;
+    bool check_encoding_match;
+    bool global;
+    const char  *item_name;
+    const char  *display_name;
+} analyzer_t;
+
+analyzer_t AnalyzerItems[] = {
+    /* request keywords */
+    { 0, 0,  true, false, "http_uri", "http uri" },
+    { 0, 0,  false, false, "http_raw_uri", "http raw uri" },
+    { 0, 0,  false, true, "http_method", "http method" },
+    { 0, 0,  false, false, "http_request_line", "http request line" },
+    { 0, 0,  false, false, "http_client_body", "http client body" },
+    { 0, 0,  true, false, "http_header", "http header" },
+    { 0, 0,  false, false, "http_raw_header", "http raw header" },
+    { 0, 0,  true, false, "http_cookie", "http cookie" },
+    { 0, 0,  false, false, "http_user_agent", "http user agent" },
+    { 0, 0,  false, false, "http_host", "http host" },
+    { 0, 0,  false, false, "http_raw_host", "http raw host" },
+    { 0, 0,  false, false, "http_accept_enc", "http accept enc" },
+    { 0, 0,  false, false, "http_referer", "http referer" },
+    { 0, 0,  false, false, "http_content_type", "http content type" },
+    { 0, 0,  false, false, "http_header_names", "http header names" },
+
+    /* response keywords not listed above */
+    { 0, 0,  false, false, "http_stat_msg", "http stat msg" },
+    { 0, 0,  false, false, "http_stat_code", "http stat code" },
+    { 0, 0,  false, true, "file_data", "http server body"},
```

Comment:
file_data can also refer to smb file xfer, nfs file xfer, smtp attachments

Reply:
In this module, it's only be used as an alias for the http server body buffer.